database/src/main/kotlin/no/nav/su/se/bakover/database/grunnlag/FormuegrunnlagPostgresRepo.kt

```diff
@@ -19,11 +18,11 @@ internal class FormuegrunnlagPostgresRepo() {
     internal fun lagreFormuegrunnlag(
         behandlingId: UUID,
         formuegrunnlag: List<Formuegrunnlag>,
-        tx: TransactionalSession,
+        session: Session,
```

Comment:
```suggestion
        session: TransactionalSession,
```

Stemmer for at det ikke er frivillig om man skal bruke Session eller TransactionalSession i de tilfellene vi ønsker transaksjoner.